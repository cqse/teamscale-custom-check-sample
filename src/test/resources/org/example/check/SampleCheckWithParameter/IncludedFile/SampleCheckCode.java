package org.example.check.SampleCheckWithParameter.IncludedFile;

public class SampleCheckCode {

	private void foo(int a) {
	}

	private void bar(int a, int b) {
	}
}
