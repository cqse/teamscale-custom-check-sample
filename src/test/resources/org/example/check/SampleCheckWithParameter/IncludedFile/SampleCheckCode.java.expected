Offsets 129-162 (lines 8-8): This is a sample finding for method bar, in a file whose name doesn't match .*Test.java$.
Offsets 98-124 (lines 5-5): This is a sample finding for method foo, in a file whose name doesn't match .*Test.java$.
